src/app/functions/server/dashboard.ts

```diff
@@ -28,6 +28,7 @@ export interface DashboardSummary {
     securityQuestions: number;
```

Comment:
@codemist We can achieve the same with something simpler, see this other PR: 
https://github.com/mozilla/blurts-server/pull/3260

I think we can get rid of the changes in this `dashboard.ts` and `DashboardTopBanner.tsx` here